content/en/continuous_integration/intelligent_test_runner/_index.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Intelligent Test Runner
+kind: documentation
+is_beta: true
+further_reading:
+  - link: "https://www.datadoghq.com/blog/streamline-ci-testing-with-datadog-intelligent-test-runner/"
+    tag: "Blog"
+    text: "Streamline CI testing with Intelligent Test Runner"
+  - link: "https://www.datadoghq.com/blog/monitor-ci-pipelines/"
+    tag: "Blog"
+    text: "Monitor all your CI pipelines with Datadog"
+---
+{{< beta-callout url="https://docs.google.com/forms/d/e/1FAIpQLSdz9w6cIC-rhHz5kU-i2aTjVa2H1lH54lM7rhjPx-8viNfdDw/viewform?usp=pp_url" d-toggle="modal" d_target="#signupModal" custom_class="sign-up-trigger">}}
+Intelligent Test Runner for CI Visibility is in private beta. You can request access by completing this form.
+{{< /beta-callout >}}
+
+{{< img src="continuous_integration/itr_overview.png" alt="Intelligent test runner enabled in test service settings in the CI section of Datadog.">}}
```

Comment:
How do you feel about moving this screenshot after the Overview H2 or even after the intro sentence? Something about the beta banner being right next to the image looks off to me.

Reply:
This is fair. In a non-beta page it might've looked more consistent with other pages but you're right it kinda looks awkward